apps/ops/api/adhoc.py

```diff
@@ -28,7 +29,7 @@ def allow_bulk_destroy(self, qs, filtered):
     def check_object_permissions(self, request, obj):
         if request.method != 'GET' and obj.creator != request.user:
             self.permission_denied(
-                request, message={"detail": "Deleting other people's script is not allowed"}
+                request, message={"detail": _("Deleting other people's script is not allowed")}
             )
         return super().check_object_permissions(request, obj)
 
```

Comment:
This text appears to contain a section that talks about checking bulk destroy functionality for JMSBulkModelViewSet and handling specific permissions and messages appropriately based on different scenarios. The content is in C++ style code which uses Django ORM.

However, it seems like there may be an oversight regarding the Python syntax used here (specifically `from django.db.models import Q`) versus the actual language being utilized - the example provided does not match standard Python usage. 

It might be more accurate to ask directly whether this block of code has any errors with regards to modern programming practices and data structure considerations. If you would like assistance with those kinds of questions please let me know!